asabiris/orchestration/sendslack.py

```diff
@@ -46,7 +46,14 @@ async def send_to_slack(self, msg):
 
 		# templates must be stores in /Templates/Slack
 		if not template.startswith("/Templates/Slack/"):
-			raise PathError(use_case='Slack', invalid_path=template)
+			raise ASABIrisError(
+				ErrorCode.INVALID_PATH,
+				tech_message="Incorrect template path '{}'. Move templates to '/Templates/Slack/".format(template),
+				error_i18n_key="Incorrect template path '{{incorrect_path}}'. Please move your templates to '/Templates/Slack/",
```

Comment:
'/Templates/Slack/' - closing quotation mark (if it goes to the user someday it will be seen)

Reply:
@language-shprt sure. Done